test/nuts/specialTypes/translation.nut.ts

```diff
@@ -161,24 +162,76 @@ describe('translations', () => {
       });
 
       describe('individual type retrieves', () => {
-        it('can retrieve COT', async () => {
+        it('can retrieve COT from CFT', async () => {
           execCmd(`project retrieve start -d ${translationPath} --json`, {
             ensureExitCode: 0,
           });
         });
 
         it('can retrieve COT from directory', async () => {
-          execCmd(
-            `project retrieve start -d ${path.join(
-              translationPath,
-              'customObject__c-es.objectTranslation-meta.xml'
-            )} --json`,
-            {
-              ensureExitCode: 0,
-            }
+          // the .objectTranslation is in my-app
+          const objectTranslationPath = path.join(
+            session.project.dir,
+            'my-app',
+            'main',
+            'default',
+            'objectTranslations',
+            'customObject__c-es',
+            'customObject__c-es.objectTranslation-meta.xml'
           );
+
+          execCmd(`project retrieve start -d ${objectTranslationPath} --json`, {
+            ensureExitCode: 0,
+          });
         });
       });
+
+      it('will retrieve across MPDs', async () => {
+        // read the file before deleting it to restore it after this test
+        const objectTranslationMyAppPath = path.join(
+          session.project.dir,
+          'my-app',
+          'main',
+          'default',
+          'objectTranslations',
+          'customObject__c-es',
+          'customObject__c-es.objectTranslation-meta.xml'
+        );
+        const objectTranslationContent = await fs.promises.readFile(objectTranslationMyAppPath, 'utf8');
+
+        // delete the file
+        await fs.promises.unlink(objectTranslationMyAppPath);
+        execCmd('project retrieve start -m CustomObjectTranslation --json', { ensureExitCode: 0 });
+        expect(
+          fs.existsSync(
+            path.join(
+              session.project.dir,
+              'force-app',
+              'main',
+              'default',
+              'objectTranslations',
+              'customObject__c-es',
+              'customObject__c-es.objectTranslation-meta.xml'
+            )
+          )
+        ).to.be.true;
```

Comment:
This isn't really retrieving across package dirs.  It's retrieving a COT to the default location.  I think we need a test that has COT and COFT and ensure it can be retrieved to the default package and to a specific non-default package.  E.g., having a COT in my-app, deploying a COFT for that COT from outside of the project, then retrieving it.